modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc

```diff
@@ -132,6 +132,14 @@ calculate accordingly.
 
 == PHP-FPM
 
+Please note that the ownCloud QA team is using solely mod_php for testing. 
```

Comment:
Please note that the ownCloud QA team is using solely mod_php for testing.

Note that `mod_php` is used exclusively in the development and QA process of the ownCloud server.